.travis.yml

```diff
@@ -18,22 +18,26 @@ addons:
 install:
   - ""
 
+# JDKs used in builds must be supported by Tycho. Check https://wiki.eclipse.org/Tycho/Release_Notes to find supported versions.
 matrix:
   fast_finish: true
   include:
+    # JDK 8 (oldest supported Java version of eclipse-cs)
     - jdk: openjdk8
       env:
         - DESC="install (openjdk8)"
         - CMD="mvn install && git diff"
 
+    # JDK 11 (Long Term Support version)
```

Comment:
JDK 11 (Long Term Support version , our next minimal version)

Reply:
I've not changed that one. I like documentation to reflect the current state, instead of creating confusion by documenting 2 different "minimal versions".

Reply:
ok, np